.github/workflows/release-create-hotfix.yml

```diff
@@ -15,17 +27,31 @@ jobs:
         with:
           app_id: ${{ secrets.GH_BOT_APP_ID }}
           private_key: ${{ secrets.GH_BOT_PRIVATE_KEY }}
+
+      - uses: actions/checkout@v4
+        with:
+          ref: develop
+          sparse-checkout: tools/actions/composites
+
+      - name: Generate ref/tag version to use during main checkout
+        uses: ./tools/actions/composites/generate-tag
+        id: format-app-tag
+        with:
+          tag_version: ${{ github.event.inputs.tag_version }}
+          application: ${{ github.event.inputs.application }}
+        
       - uses: actions/checkout@v4
         with:
-          ref: main
+          ref: ${{ steps.format-app-tag.outputs.main_ref }}
           token: ${{ steps.generate-token.outputs.token }}
+
       - name: Setup git user
-        uses: LedgerHQ/ledger-live/tools/actions/composites/setup-git-user@develop
+        uses: ./tools/actions/composites/setup-git-user
       - name: create hotfix branch
         run: |
           git checkout -b hotfix
       - name: Setup the toolchain
-        uses: LedgerHQ/ledger-live/tools/actions/composites/setup-toolchain@develop
+        uses: ./tools/actions/composites/setup-toolchain
```

Comment:
you may need to use directly (if possible) `setup-caches` instead

Reply:
there's a change in the setup-caches PR for this. At this rate, that PR will be merged after this one